index.js

```diff
@@ -45,7 +45,7 @@ function createFunction(parameters, defaultType) {
                 featureFunctionStops.push([featureFunctions[z].zoom, createFunction(featureFunctions[z])]);
             }
             fun = function(zoom, feature) {
-                return evaluateExponentialFunction({ stops: featureFunctionStops, base: parameters.base }, zoom)(zoom, feature);
+                return evaluateExponentialFunction({ stops: featureFunctionStops, base: parameters.base, overflow: parameters.overflow }, zoom)(zoom, feature);
```

Comment:
I'm 👎  on the name "overflow"

 > **overflow** the generation of a number or some other data item that is too large for an assigned location or memory space.

The term that is typically used for this concept is "clamp" (for example [c++ std::clamp](http://en.cppreference.com/w/cpp/algorithm/clamp), [lodash clamp](https://lodash.com/docs/4.16.3#clamp), [`Uint8ClampedArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8ClampedArray))